dev/SapphireTest.php

```diff
@@ -30,12 +28,7 @@ class SapphireTest extends PHPUnit_Framework_TestCase {
 	 * @var FixtureFactory
 	 */
 	protected $fixtureFactory;
-	
-	/**
-	 * @var bool Set whether to include this test in the TestRunner or to skip this.
-	 */
-	protected $skipTest = false;
```

Comment:
I know this references TestRunner, but are you 100% sure this no longer has any use? Does markTestSkipped skip the entire class or just the one test method?